docs/blog/2018-03-07-why-we-created-the-plugin-library/index.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Why we created the Gatsby plugin library
+date: 03-07-2018
+author: Shannon Soper
+---
+
+## Why build a plugin / package library
+
+In over 60 interviews with Gatsby users, plugin discoverability appeared as a top pain point (see [blog post on the Gatsby UX research program](/2017-12-20-introducing-the-gatsby-ux-research-program/) for a full list of pain points). So we built a plugin / package library [here](/packages/)!
```

Comment:
Should be active voice & give timeline context, eg, "from October 2017 to Feb 2018, we conducted..."